### ISSUE TITLE
Jeté Battu and Fouetté Battu

### ISSUE THREAD

**Issue by rsryman** [rryman/BeneshForBalletBook3#17]
See 2 animations uploaded to Dropbox "For Robyn." Rhonda added the red foot, and also fixed the heads in the back view (I had forgotten to add a slight "raise" so the head doesn't appear dropped).
**Animations added to book 30/11/2015.**

Ben ends on a bent leg, so Robyn please add a bent knee sign to Bar 4 ct 1, yes? (Monica's book is SOOO confusing on this point.)
**Bent knee added in both solid and part-bar examples.**
**Do you want the end of the "i" box info changed? If so, how?:
 "Bar 16: Finish last jeté battu en fondu? in demi-plié? with bent knee? and hold? "**

The BMN shows 3 alternating repeats (16 bars) for the solid bar repeat example but an other side repeat (8-bars) for the dotted part-bar repeat option. Did you intend this? If so, do you need 2 shortened animations?
**I just cut and pasted the notation we had in the original Book 2 with Repeats to mock-up Book 3. We had used the part-bar example in the Appendix and had notated this enchainement out in full to show the different options. It took up too much space on the page to write out 4 times in full, so we used notation and animation with only repeat on other side. In the main chapter of the book, we had Ben repeating 3 times.**
**I am still not sure on our strategy. I am assuming we are not giving examples with the notation in full for this section of the book? We will only do that for the Benesh Basics chapter? If that's the case, we are safe to have Ben perform this enchainement 4 times in all. Therefore, I have updated the part-bar notation.**

Very clear text for the dotted part-bar option!!

**Comment by rsryman** [comment]
Re i-icon text (in line with the Cecchetti Dictionary): 
"Bar 16: Finish last jeté battu en fondu and hold. "

I see you have added an explanatory note at the bottom left of page 58. As we complete the lead-up to this page we may decide if it is necessary, but leave for now. We may want use pop-ups to reiterate theory.

**Comment by rryman** [comment]
Rhonda, I had to take all the pieces from the old Book 2 with the Repeats and move all the info around. Some pieces fit and some pieces didn't. I didn't want to lose the pieces that didn't quite fit anywhere, so I plonked them on pages in case we needed to use them somewhere else. They are only there as a reminder to me as we work our way through the book. I will make sure I remove them all before we publish Book 3!
